tensorboard/webapp/header/reload_container.ts

```diff
@@ -13,15 +13,28 @@ See the License for the specific language governing permissions and
 limitations under the License.
 ==============================================================================*/
 import {Component} from '@angular/core';
-import {Store} from '@ngrx/store';
+import {Store, createSelector} from '@ngrx/store';
 import {Observable} from 'rxjs';
-import {map} from 'rxjs/operators';
+import {map, withLatestFrom} from 'rxjs/operators';
 
 import {State} from '../core/store/core_types';
-import {getPluginsListLoaded} from '../core/store/core_selectors';
+import {
+  getPlugins,
+  getActivePlugin,
+  getPluginsListLoaded,
+} from '../core/store/core_selectors';
 import {manualReload} from '../core/actions';
 import {DataLoadState} from '../types/data';
 
+const isReloadDiabledByPlugin = createSelector(
+  getPlugins,
+  getActivePlugin,
+  (plugins, id) => {
+    if (!id || !plugins || !plugins[id]) return false;
```

Comment:
I think `plugins` starts as an `{}` and is never non-truthy?  Is it safe to drop `!plugins`?